alg/sha256_shani.h

```diff
@@ -10,7 +10,12 @@
  * instructions, and should only be used if CPUSUPPORT_X86_SHANI and _SSSE3
  * are defined and cpusupport_x86_shani() and _ssse3() return nonzero.
  */
+#ifdef POSIXFAIL_ABSTRACT_DECLARATOR
```

Comment:
I'm not certain whether you'd prefer `#ifdef` or `#ifndef`.

... of course you'd prefer none at all, because this looks disgusting (especially in a header file!).  But if we want to support the default compiler on `configure` on OpenBSD, we need something like this.  :(

Reply:
`#ifdef` is fine here.